### DISCUSSION TITLE
Change default drive for installed apps

### DISCUSSION THREAD

**Discussion by roycerezendes** [ChrisTitusTech/winutil#2803] [Q&A]
I like to install my apps on a different drive than my OS, could you add an option that let's me choose a default drive that all these applications install to?  Also along those lines does it already check other drives for already installed apps?  Thanks for the amazing tool.

**Comment by DataGoblin** [comment]
Temp solution could be this setting in windows itself
![image](https://github.com/user-attachments/assets/969f7bfc-839a-41a6-9928-ec189f643920)

**Comment by roycerezendes** [comment]
Thanks for the quick response!  I actually found out about that setting
After I posted the question and it turns out that setting apparently only
affects Microsoft store apps?  Didn't change regular programs for me, then
I found some windows registry entries to change and that did the trick.

**Reply by WrayOfSunshine** [reply]
What registry entries did you change?

**Comment by roycerezendes** [comment]
It was a while ago I would have to search for them again but I believe it
was two entries and it just changes the default program install location.
